packages/cli/src/commands/init.ts

```diff
@@ -135,8 +135,8 @@ export async function promptForConfig(
     {
       type: "text",
       name: "tailwindConfig",
-      message: `Where is your ${highlight("tailwind.config.js")} located?`,
-      initial: defaultConfig?.tailwind.config ?? DEFAULT_TAILWIND_CONFIG,
+      message: `Where is your ${defaultConfig?.tsx ? highlight("tailwind.config.ts") : highlight("tailwind.config.js")} located?`,
```

Comment:
Hey @Jacksonmills, I was also checking to do something like this just now and I saw your PR. 

What I've tried was convert the message to a function which you can: https://www.npmjs.com/package/prompts, and then according to previous value of typescript, update the suggested path as ts or js.

```
{
  type: 'text',
  name: 'tailwindConfig',
  message: (prev, values) => `Where is your ${highlight(`tailwind.config.${values.typescript ? 'ts' : 'js'}`)} located?`
}
```

Altho, I don't know if this would work with other selected options? @shadcn what do you think?

Reply:
Nice, waiting on hearing from shad on this one as they mentioned doing all of this automatically soon